openid-connect-4-credential-issuance-1_0.md

```diff
@@ -302,7 +326,7 @@ Below is a non-normative example of a `claims` parameter:
 ```json=
 {
     "credential": {
-        "type": ["uri":"..."],
+        "type": ["uri":"https://did.example.org/healthCard","https://did.exmaple.org/mDL"],
```

Comment:
Should the type property be:
(openapi schema definition)
credential:
  title: credential
  description: Top-level claims parameter for requesting credentials
  type: object
  properties:
    type:
      description: A list of requested verifiable credential schemas
      type: array
      items:
        type: object
        properties:
          uri:
            description: URI to the Verifiable Credential schema
            type: string
            format: string
    format:
      description: Format in which the VCs should be issued. It MUST be jwt_vc or ldp_vc.
      type: string

example:
"credential": {
        "type": [{"uri":"https://did.example.org/healthCard"},{"uri":"https://did.exmaple.org/mDL"}],
        "format": "jwt_vc"
}

Reply:
Add a note: "How to submit input presentations"
when sending VCs as input in the authz req - reusing VP_token OR mandating PAR?

change the structure, not per flow, but per endpoint. and illustrate flow in the examples. - look at OAuth 2.1

Reply:
I think we need a structure where the format can be determined per credential type.

Reply:
Look at the example, the mDL credential will most likely not be provided as ldp_vc.

Reply:
It's also inline with the desired PE syntax changes.

Reply:
added description of vp_token and PAR